sshcommand

```diff
@@ -58,6 +58,12 @@ case "$1" in
 
     getent passwd "$USER" > /dev/null || false
     USERHOME=$(sh -c "echo ~$USER")
+
+    NEW_KEY=$(grep "NAME=\\\\\"$NAME"\\\\\" "$USERHOME/.ssh/authorized_keys" || true)
+    if [ ! -z "$NEW_KEY" ]; then
```

Comment:
Should we show an error here (to stderr) telling the user that the name for the key has already been taken, and that they should use another name?